docs/Running_Occlum_with_Docker_and_OCI_Runtime_rune.md

```diff
@@ -55,39 +56,25 @@ This user guide provides the steps to run Occlum with Docker and OCI Runtime `ru
     3. Update the apt and install the packages.
     ```shell
     apt-get update
-    apt-get install rune occlum-pal
+    apt-get install rune occlum
+    source /etc/profile
```

Comment:
Why source this file?

Reply:
To make the new installed occlum binaries and libraries work

Reply:
That is OK.